src/widget/OktaSignIn.js

```diff
@@ -117,9 +116,15 @@ var OktaSignIn = (function () {
     require.config(requireConfig);
     OktaAuth = require('vendor/OktaAuth');
     Util = require('util/Util');
+    LoginRouter = require('LoginRouter');
+
 
     authClient = new OktaAuth({ uri: options.baseUrl, transformErrorXHR: Util.transformErrorXHR });
-    _.extend(this, getProperties(authClient, options));
+    _.extend(this, LoginRouter.prototype.Events, getProperties(authClient, options, LoginRouter));
```

Comment:
nit: Switch order of passing in options and LoginRouter (options is usually the last argument)